ssl/record/ssl3_record.c

```diff
@@ -711,16 +711,20 @@ int ssl3_get_record(SSL *s)
             goto end;
         }
 
+        /*
+         * Usually thisrr->length is accurate, just not when KTLS receives
+         * adjacent maximum-size packets.
+         * Just assume the SSL3_RT_MAX_PLAIN_LENGTH is checked in the kernel.
+         */
         if (thisrr->length > SSL3_RT_MAX_PLAIN_LENGTH
```

Comment:
I added a trace here, when thisrr->length >= SSL3_RT_MAX_PLAIN_LENGTH,
and print the lengths at this statement.
first large packet always was 0x4000, followed by a couple 0x4130